src/main/webapp/WEB-INF/tags/instructor/instructorPage.tag

```diff
@@ -0,0 +1,20 @@
+<%@ tag description="Generic Instructor Page" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+<%@ taglib tagdir="/WEB-INF/tags" prefix="t" %>
+<%@ taglib tagdir="/WEB-INF/tags/instructor" prefix="ti" %>
+<%@ attribute name="pageTitle" required="true" %>
+<%@ attribute name="bodyTitle" required="true" %>
+<t:page pageTitle="${pageTitle}" bodyTitle="${bodyTitle}">
+    <jsp:attribute name="navBar">
+        <ti:nav />
+    </jsp:attribute>
+    <jsp:attribute name="headerJs">
+        <script type="text/javascript" src="/js/instructor.js"></script>
+        <script type="text/javascript" src="/js/instructorHome.js"></script>
+        <script type="text/javascript" src="/js/ajaxResponseRate.js"></script>
+        <script type="text/javascript" src="/js/instructorFeedbackAjaxRemindModal.js"></script>
```

Comment:
Rather, they should be injected to instructorHome.jsp only. Not sure if the current structure allows injecting stuff to the `<head>` section of a specific jsp page though.